sentry-spring/src/test/kotlin/io/sentry/spring/SentryCheckInAdviceTest.kt

```diff
@@ -0,0 +1,173 @@
+package io.sentry.spring
+
+import io.sentry.CheckIn
+import io.sentry.CheckInStatus
+import io.sentry.IHub
+import io.sentry.SentryOptions
+import io.sentry.protocol.SentryId
+import io.sentry.spring.checkin.SentryCheckIn
+import io.sentry.spring.checkin.SentryCheckInAdviceConfiguration
+import io.sentry.spring.checkin.SentryCheckInPointcutConfiguration
+import org.junit.jupiter.api.assertThrows
+import org.junit.runner.RunWith
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.EnableAspectJAutoProxy
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig
+import org.springframework.test.context.junit4.SpringRunner
+import kotlin.RuntimeException
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+@RunWith(SpringRunner::class)
+@SpringJUnitConfig(SentryCheckInAdviceTest.Config::class)
+class SentryCheckInAdviceTest {
+
+    @Autowired
+    lateinit var sampleService: SampleService
+
+    @Autowired
+    lateinit var sampleServiceNoSlug: SampleServiceNoSlug
+
+    @Autowired
+    lateinit var sampleServiceHeartbeat: SampleServiceHeartbeat
+
+    @Autowired
+    lateinit var hub: IHub
+
+    @BeforeTest
+    fun setup() {
+        whenever(hub.options).thenReturn(SentryOptions())
+    }
+
+    @Test
+    fun `when class is annotated with @SentryCheckIn, every method call creates two check-ins`() {
```

Comment:
Should we change the wording here to
```suggestion
    fun `when method is annotated with @SentryCheckIn, every method call creates two check-ins`() {
```